lib/agent.js

```diff
@@ -257,7 +257,7 @@ Agent.prototype._sendOp = function(collection, id, op) {
     src: op.src,
     seq: op.seq
   };
-  if (op.op) message.op = op.op;
+  if ('op' in op) message.op = op.op;
```

Comment:
Wait, so `op` here can be `null`, right?

But, `in` doesn't work on `null`...

![image](https://user-images.githubusercontent.com/68416/85033623-2a64c000-b14f-11ea-8bc7-83f5f55113d7.png)

In any case, since the solution will need to be duplicated so many places across the codebase, it might make sense to introduce a helper function, like `hasOp(op)`, so that the actual checking logic could be modified in just one place.

Reply:
Here `op` will always be an object I think. (The old condition was based on this too). So more

<img width="555" alt="Screenshot 2020-06-18 at 16 43 51" src="https://user-images.githubusercontent.com/1465371/85034948-ec818f00-b182-11ea-87c5-3112d487798e.png">

Reply:
Yep, @antoinelyset is right here, `op` is an object. It's `op.op` in this code that might be `null`